sound/soc/sof/intel/pci-tgl.c

```diff
@@ -65,7 +65,7 @@ static const struct sof_dev_desc ehl_desc = {
 	.default_tplg_path = "intel/sof-tplg",
 	.default_fw_filename = "sof-ehl.ri",
 	.nocodec_tplg_filename = "sof-ehl-nocodec.tplg",
-	.ops = &sof_cnl_ops,
```

Comment:
Oh I also noticed you have the "intel" in the prefix for EHL but not for the others. Can you please make it consistent if you're adding the prefix here then add it for all with an upper case "I" for Intel?

Reply:
> Oh I also noticed you have the "intel" in the prefix for EHL but not for the others. Can you please make it consistent if you're adding the prefix here then add it for all with an upper case "I" for Intel?

@ranj063 Thanks. I will add the fixes tag and use "Intel" prefix for the patches.